salt/srv/salt/base/calamari-crush-location.py

```diff
@@ -20,14 +20,14 @@ def get_last_crush_location(osd_id):
     key = 'daemon-private/osd.%s/v1/calamari/osd_crush_location' % osd_id
     osd_keyring = '/var/lib/ceph/osd/ceph-%s/keyring' % osd_id
     admin_keyring = '/etc/ceph/ceph.client.admin.keyring'
-    commands = (['sudo', 'ceph', '--name', 'osd.%s' % osd_id, '--keyring', osd_keyring, 'config-key', 'get', key],
+    commands = (['ceph', '--name', 'osd.%s' % osd_id, '--keyring', osd_keyring, 'config-key', 'get', key],
```

Comment:
...the same issue may exist with the monitoring code in ceph.py, but that would just partially trip up monitoring, whereas this starts to interfere with the operation of the system